examples/mobile/UIComponents/components/dialogs/picker.html

```diff
@@ -93,6 +98,16 @@ <h1>
 					<button class="ui-btn" data-style="flat">Done</button>
 				</div>
 			</div>
+			<div class="ui-popup" id="date-time-picker-wheel">
+				<div class="ui-popup-content ui-popup-has-date-picker">
+					<div class="ui-datetime-picker" data-view="wheel" data-format="12" data-value="2020-03-12"></div>
```

Comment:
Does only set the date through "data-value"? can't set the time?
And just question, if we can set the date, the format is 2020-03-12 not like 2020:03:12?

Reply:
Thank you for review. Work is in progress now, but you are right. I'll do it for accept the same string value as `Data` constructor.
```
let birthday = new Date('December 17, 1995 03:24:00')
let birthday = new Date('1995-12-17T03:24:00')
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date

or better the API should be the same as:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local